### ISSUE TITLE
Korvi not found/published with Korge 3.0.0-Beta*

### ISSUE THREAD

**Issue by zhanghai** [korlibs/korge#795]
There is `KorgeExtension.supportVideo()`, but using it results in a dependency resolution failure. I couldn't find `korvi` under `korge2` on MavenCentral either.

**Comment by zhanghai** [comment]
Could you elaborate on the reason for replacing it with a source code dependency? Playing videos may still be an important part of a game (not all games can use real time CG to achive the same effect as prerendered videos), so it would be nice if it can still be an easy-to-integrate official binary dependency of Korge.

**Comment by zhanghai** [comment]
I see. I'm still new to KMM and couldn't understand why local compilation will require less effort than the same code being precompiled (assuming the files/artifacts for each platform are built and uploaded by scripts), but I guess you know this far better than me. And glad to hear that it will be the same for us consumers, so thanks for the clarification and the work on this project!

**Comment by zhanghai** [comment]
Note that I was able integrate Korvi with Korge 3.0.0 Beta by manually specifying the dependency:

```
dependencyMulti("com.soywiz.korlibs.korvi:korvi:${BuildVersions.KORGE}", registerPlugin = false)
```

Korvi works fine in the browser, but when I was playing a 2K 60FPS MP4 file on JVM (on a reasaonbly performant desktop) it gives me only 8 FPS and no sound. I assume the reason is that Korvi on JVM is using a software decoder and doesn't support AAC audio. It seems to me we almost have to use hardware decoding nowadays to support modern videos, so I guess I may need to find something else for videos 🤔

**Comment by soywiz** [comment]
Yeah, on the JVM it uses a JVM library for mp4 decoding. Korvi is pretty experimental, though it would be possible to plug libavcodec and other other libraries from ffmpeg to do that, or plug to each operating system functionality, or compile ffmpeg with wasm, and load it in the JVM.
I wanted something portable despite the performance/encoding so the initial implementation was like that. There is another thing in the roadmap that should be just a few hours of work: https://github.com/korlibs/korge/issues/535 despite probably not for 2K, and not space-efficient.
In any case updating this myself is not in my short-term roadmap; but contributions are welcome.
